lib/parse-args.js

```diff
@@ -188,6 +189,28 @@ function buildYargs (withCommands = false) {
   return yargs
 }
 
+function loadConfigFile (path, ext) {
+  let config = {}
+  const jsExts = ['.js', '.cjs']
+  const ymlExts = ['.yml', '.yaml']
+  if (jsExts.includes(ext)) {
+    config = require(path)
+  } else if (ymlExts.includes(ext)) {
+    config = require('js-yaml').load(readFileSync(path, 'utf8'))
```

Comment:
Perhaps we could add a `try/catch` here, and print a useful message that explains that `js-yaml` is necessary to handle `yaml` extensions.

Reply:
The same argument could be made for the other cases. That said I think the default error is fine:
```sh
$ node test.js 
node:internal/modules/cjs/loader:1042
  throw err;
  ^

Error: Cannot find module 'js-yaml'
Require stack:
- /tmp/test.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1039:15)
    at Module._load (node:internal/modules/cjs/loader:885:27)
    at Module.require (node:internal/modules/cjs/loader:1105:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/tmp/test.js:1:1)
    at Module._compile (node:internal/modules/cjs/loader:1218:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
    at Module.load (node:internal/modules/cjs/loader:1081:32)
    at Module._load (node:internal/modules/cjs/loader:922:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/tmp/test.js' ]
}

Node.js v18.13.0
```

Reply:
The js-yml is currently a dependency of other packages in the c8 dependency tree.  So yes I agree that we should add a try/catch, and also explicitly define the dependency in package.json.  Here is the list of packages that c8 depends on that currently rely on js-yaml

``` bin/bash
***************REDACTED********************:~/codez/tmp/c8$ grep -Rnil 'js-yaml' node_modules/ | sed 's@node_modules/@@g' | sed 's@/@ @g' | awk '{print $1}' | uniq
eslint
uri-js
.package-lock.json
.bin
diff
mocha
typescript
@eslint
js-yaml
***************REDACTED********************:~/codez/tmp/c8$
```